icons/debug.svg

```diff
@@ -0,0 +1,21 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m10 3 1 2" />
+  <path d="m14 3-1 2" />
+  <path d="M20 5c0 1.3-.6 2.4-1.6 3.2" />
+  <path d="M8 9C5.8 9 4 7.2 4 5" />
+  <line x1="4" x2="8" y1="12" y2="12" />
+  <path d="M18.4 15.8c1 .7 1.6 1.9 1.6 3.2" />
+  <path d="M4 19c0-2.2 1.8-4 4-4" />
+  <polygon points="12,15 17,12 12,9" />
+  <path d="M14.1 18.4c-.6.4-1.3.6-2.1.6h0c-2.2 0-4-1.8-4-4V9c0-2.2 1.8-4 4-4h0c.7 0 1.4.2 2 .6" />
```

Comment:
Hah, that's actually pretty neat how the play icon fits exactly within. :D

It's 1px higher up than the regular `bug` icon though:
```suggestion
  <path d="m10 4 1 2" />
  <path d="m14 4-1 2" />
  <path d="M20 6c0 1.3-.6 2.4-1.6 3.2" />
  <path d="M8 10c-2.2 0-4-1.8-4-4" />
  <path d="M4 13h4" />
  <path d="M18.4 16.8c1 .7 1.6 1.9 1.6 3.2" />
  <path d="M4 20c0-2.2 1.8-4 4-4" />
  <path d="m12 16 5-3-5-3v6Z" />
  <path d="M14.1 19.4A4.01 4.01 0 0 1 8 16v-6c0-2.2 1.8-4 4-4 .7 0 1.4.2 2 .6" />
```